### DISCUSSION TITLE
make style scoped instead of being global

### DISCUSSION THREAD

**Discussion by ninja-liu** [primefaces/community#1846] [PrimeVue] [answered]
is it possible to make styles scoped to components used instead of being global ? i mean all the styles including presets where "import 'primevue/resources/themes/aura-light-green/theme.css'" would be not needed.

**Comment by tugcekucukoglu** [comment, marked as answer]
PrimeVue now is moving a theming API. For more information https://www.youtube.com/watch?v=MYkoX6WmhiE
